src/classes/pilot/Pilot.ts

```diff
@@ -730,11 +730,12 @@ class Pilot
       mech.MechLoadoutController.UpdateLoadouts()
     })
 
-    if (sync && data.state) {
-      this._state.Update(this, data.state, sync)
-    } else {
+    // Updating active state will need to be revisited
+    //if (sync && data.state) {
+    //  this._state.Update(this, data.state, sync)
+    //} else {
```

Comment:
This block wasn't being executed anyway, so I went ahead and commented it out until we get back to an infrastructure that will allow for real active state sync.